src/Guid.ts

```diff
@@ -10,12 +10,14 @@ export class Guid implements IEquatable, IFormattable, IJsonStringifyable {
      * @remarks It is the default constructor, for creating an actual GUID
      *          you will normally use Guid.newGuid() or Guid.parse(string).
      */
-    private constructor() { }
+    private constructor(v: string) {
+        this.v = v;
+    }
 
     /**
      * The underlying value.
      */
-    private v = '00000000-0000-0000-0000-000000000000';
+    private readonly v: string; //= ;
```

Comment:
In case of naming things should this just be value.
it will be easier to review if you just change a single line in the future